vcd/datasource_vcd_nsxt_edgegateway.go

```diff
@@ -176,6 +176,11 @@ func datasourceVcdNsxtEdgeGateway() *schema.Resource {
 				Computed:    true,
 				Description: "Total number of IP addresses allocated for this Edge Gateway",
 			},
+			"uses_ip_spaces": {
```

Comment:
Is there any chance this would become writable in the future? If yes, `uses_ip_spaces` could be named `use_ip_spaces`. However, if it's read-only for the foreseeable future, then the current `uses_ip_spaces` is more descriptive.

Reply:
Changed to `uses_ip_spaces` here, resource, tests and in website docs. Test still passes :)

Reply:
I think you have meant `use_ip_spaces` - and it looks great now!